### ISSUE TITLE
Usage of LayoutLM

### ISSUE THREAD

**Issue by ingaluxemburg** [microsoft/unilm#703]
Hello. I'm trying to use LayuotLm for token classification, I need to use bboxes, and get a 13-classes clasification as an model output, but insteed i get torch.tensor with shape (1, 4, 2) when I pass "Hello", "world" with corresnding bboxes as an input. So my questions are:
1) how to get 13-class classification
2) where to find labels of classes
3) will I get results using pretrained model from HuggingFace (as i do now), or it nesessary to fine-tune?
To be absolutely clear, here is link on HuggingFace example code which I trying to use: 
https://huggingface.co/docs/transformers/model_doc/layoutlm#transformers.LayoutLMForTokenClassification

**Comment by wolfshow** [comment]
@ingaluxemburg It depends on which dataset you are working on. If you work on the FUNSD dataset, it is recommended to look at https://huggingface.co/nielsr/layoutlmv2-finetuned-funsd

**Comment by ingaluxemburg** [comment]
@wolfshow Thank you for answer. But how to use this model? I have not find any description. My goal is to take PDF, put it in a model and have 13 numbers for every token - probabilities from the paper.
